unified_runtime/server.py

```
@@ -515,6 +487,71 @@ async def trust_score(proposal: Dict[str, Any]) -> Dict[str, Any]:
+            ds_router.config.support_threshold = float(thresholds["support_threshold"])
+        if "max_cot_tokens" in thresholds:
+            ds_router.config.max_cot_tokens = int(thresholds["max_cot_tokens"])
+            
+        return {
+            "status": "updated",
+            "current_thresholds": {
+                "conf_threshold": ds_router.config.conf_threshold,
+                "support_threshold": ds_router.config.support_threshold,
+                "max_cot_tokens": ds_router.config.max_cot_tokens
+            }
+        }
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)